tests/Unit/CredentialManagers/CacheStoreTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Tests\Webfox\Xero\Unit\CredentialManagers;
+
+use Exception;
+use Illuminate\Cache\Repository;
+use Illuminate\Session\Store;
+use Illuminate\Support\Facades\Session;
+use Mockery\MockInterface;
+use Tests\Webfox\Xero\TestCase;
+use Tests\Webfox\Xero\TestSupport\Mocks\MockAccessToken;
+use Webfox\Xero\Oauth2CredentialManagers\CacheStore;
+use Webfox\Xero\Oauth2Provider;
+
+class CacheStoreTest extends TestCase
+{
+    public function test_you_can_get_array_store_without_existing_data()
```

Comment:
A couple of tests in this file need their names updated to `_cache_` instead of `_array_`. I wonder if we can use [PHP Unit Data Providers](https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers) to clear up the duplication in the tests?

Reply:
Yeah, I agree with this, I'll create a ticket for this.